_docs/aws_resources.md

```diff
@@ -144,4 +146,25 @@ def lambda_handler(event, context):
 ```
 
 
+## Reserved Words
+
+When working with tables in AWS, it's important to keep mind of a few words that should not be used. These are words in use by the database itself for a variety of different reasons, and using these terms as a primary key, or sort key, will cause errors when trying to call those columns from a JavaScript call.
+
+A few example words are:
+- LOCATION
+- ATTRIBUTE
+- BLOB
+- CLASS
+- DIAGNOSTICS
+- GROUP
+
+**DO NOT USE THESE WORDS**
+
+They will only cause problems for you down the line!
+
+Before finalizing your database schema, be sure to check ALL TERMS against these reserved words. If you use on of these terms for your partition key, or sort key, there will be no way to change them. Failure to double check may mean your group will have to terminate your table, and recreate it so it no longer uses these terms.
+
+For a full list of terms, check out the resource for DynamoDB: [Reserved Words in DynamoDB
+PDF](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/ReservedWords.html)
 
+Each AWS database service will have its own set of reserved words, so be sure to check those out early, and reference them often before finalizing your tables.
```

Comment:
Suggest removing references to "database", since services beyond DB will likely have reserved words. Small grammar fix
```suggestion
Each AWS service will have its own set of reserved words, so be sure to check them out prior to finalizing your implementation.
```